Monika After Story/game/script-topics.rpy

```diff
@@ -744,11 +745,11 @@ label mas_rerandom:
 
                 #Run the rerandom callback function
                 rerandom_callback_data = rerand_ev.rules.get("rerandom_callback", None)
-                if isinstance(rerandom_callback_data, dict) and callable(rerandom_callback_data["function"]):
+                if rerandom_callback_data is not None:
                     try:
-                        rerandom_callback_data["function"](
-                            *rerandom_callback_data.get("args", list()),
-                            **rerandom_callback_data.get("kwargs", dict())
+                        rerandom_callback_data.callable(
+                            *rerandom_callback_data.args,
+                            **rerandom_callback_data.kwargs
                         )
```

Comment:
```suggestion
                        rerandom_callback_data()
```
Curry implements `__call__` so you can call directly. The args used when calling renpy.partial are passed into the callable already.